crates/serpen/tests/fixtures/ast_rewriting/regular_import_aliases/utils/helpers.py

```
@@ -0,0 +1,16 @@
+"""Utility module for testing regular import aliases."""
+
+
+def helper_function():
+    """A simple helper function."""
+    return "helper_result"
+
+
+class UtilityClass:
+    """A simple utility class."""
+
+    def __init__(self, value):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)